lib/web_ui/lib/src/engine/semantics/incrementable.dart

```diff
@@ -45,11 +45,11 @@ class Incrementable extends RoleManager {
     _element.setAttribute('role', 'slider');
 
     _element.addEventListener('change', (_) {
-      if (_element.disabled) {
+      if (_element.disabled!) {
```

Comment:
Curious why this property is nullable. Even the most ancient browsers should have this property, no?

Reply:
Good question. Here, it's a limitation of the language: there is a class that extends `InputElement` that has `disabled` marked as nullable (probably due to some incompatibility), so this `InputElement.disabled` needs to also be nullable.